web/src/user_pill.js

```diff
@@ -109,8 +109,9 @@ export function has_unconverted_data(pill_widget) {
 }
 
 export function typeahead_source(pill_widget) {
-    const persons = people.get_realm_users();
-    return filter_taken_users(persons, pill_widget);
+    // includes ONLY humans excluding bots
+    const humans = people.get_realm_humans();
+    return filter_taken_users(humans, pill_widget);
```

Comment:
This change modifies ALL pill typeaheads, not just the custom profile fields one. I believe the `web/src/add_subscribers_pill.js` call to `pill_typeahead.set_up` does need to support bots, so this will break that widget.

What I think you want is to add an option when initializing a pill typeahead widget for whether or not to include bots.